doc/docs/master/deploy-manage/manage/s3gateway/supported-operations.md

```diff
@@ -0,0 +1,459 @@
+the MinIO client commands.
+The S3 bucket that you create is a branch in a repository
+in Pachyderm.
+
+To create an S3 bucket, complete the following steps:
+
+1. Use a corresponding command below to create a new
+S3 bucket, which is a repository with a branch in Pachyderm.
+
+   * If you are using MinIO, type:
+
+     ```bash
+     mc mb local/master.test
+     ```
+
+     **System Response:**
+
+     ```
+     Bucket created successfully `local/master.test`.
+     ```
+
+   * If you are using AWS, type:
+
+     ```bash
+     aws --endpoint-url http://localhost:30600/ s3 mb s3://master.test
+     ```
+
+     **System Response:**
+
+     ```
+     make_bucket: master.test
+     ```
+
+   * If you are using S3cmd, type:
+
+     ```bash
+     s3cmd mb s3://master.test
+     ```
+
+     This command creates the `test` repository with the `master` branch.
+
+1. Verify that the S3 bucket has been successfully created:
+
+   * If you are using MinIO, type:
+
+     ```bash
+     mc ls local
+     ```
+
+     **System Response:**
+
+     ```
+     [2019-07-18 13:32:44 PDT]      0B master.test/
+     [2019-07-12 15:09:50 PDT]      0B master.train/
+     [2019-07-12 14:58:50 PDT]      0B master.pre_process/
+     [2019-07-12 14:58:09 PDT]      0B master.split/
+     [2019-07-12 14:58:09 PDT]      0B stats.split/
+     [2019-07-12 14:36:27 PDT]      0B master.raw_data/
+     ```
+
+   * If you are using AWS, type:
+
+     ```bash
+     aws --endpoint-url http://localhost:30600/ s3 ls
+     ```
+
+     **System Response:**
+
+     ```
+     2019-07-26 11:35:28 master.test
+     2019-07-12 14:58:50 master.pre_process
+     2019-07-12 14:58:09 master.split
+     2019-07-12 14:58:09 stats.split
+     2019-07-12 14:36:27 master.raw_data
+          ```
+   * If you are using S3cmd, type:
+
+     ```bash
+     s3cmd ls
+     ```
+
+     **System Response:**
+
+     ```
+     2019-07-26 11:35 master.test
+     2019-07-12 14:58 master.pre_process
+     2019-07-12 14:58 master.split
+     2019-07-12 14:58 stats.split
+     2019-07-12 14:36 master.raw_data
+     ```
+
+   * You can also use the `pachctl list repo` command to view the
+   list of repositories:
+
+     ```bash
+     pachctl list repo
+     ```
+
+     **System Response:**
+
+     ```
+     NAME               CREATED                    SIZE (MASTER)
+     test               About an hour ago          0B
+     train              6 days ago                 68.57MiB
+     pre_process        6 days ago                 1.18MiB
+     split              6 days ago                 1.019MiB
+     raw_data           6 days ago                 2.561MiB
+     ```
+
+     You should see the newly created repository in this list.
+
+### Delete an S3 Bucket
+
+You can delete an empty S3 bucket in Pachyderm by running a corresponding
+command for your S3 client. The bucket must be empty and have no versioned
+objects. You can also use the `--force` flag to delete all objects in the
```

Comment:
1) The bucket just needs to be empty, but there's no requirement about no versioned objects
2) The `--force` flag is only supported in some CLIs, so not sure if should go here

Reply:
> Deletes an empty S3 bucket. A bucket must be completely empty of objects and versioned objects before it can be deleted. 

from here: https://docs.aws.amazon.com/cli/latest/reference/s3/rb.html

Reply:
That's true for s3, not s3 gateway

Reply:
removed